inclusion_connect/users/models.py

```diff
@@ -52,6 +57,20 @@ def id(self):
     def must_accept_terms(self):
         return self.terms_accepted_at is None or self.terms_accepted_at < settings.NEW_TERMS_DATE
 
+    def set_next_redirect_uri(self, next_redirect_uri):
```

Comment:
Indique que la méthode appelle `.save()` et non `setattr`.
```suggestion
    def save_next_redirect_uri(self, next_redirect_uri):
```

Reply:
Objectif bonus, c’est un peu dommage de faire le `.save()` dans cette méthode. Par exemple, dans `RegisterForm.save()`, on pourrait éviter le `.save()` supplémentaire en définissant les valeurs sur l’objet utilisateur avant d’appeler `super().save()`.
Après, c’est discutable car ça enterre un peu la sauvegarde du next URL pour éviter une requête en doublon.
Donc comme tu veux.

Reply:
Je me suis dit qu'on n'était pas à ça près en terme de requête en DB,
Et j'ai trouvé plus simple de le faire sur l'objet pour être sur d'avoir toujours le même comportement.